### PR TITLE
Superwise - Datadog Integration

### PR DESCRIPTION
**Pull request by oryan-omer**
### What does this PR do?
This PR adds Superwise (Superwise is a platform for model observability of high-scale ML operations) as an integration in  Datadog.

### Motivation
Creating the integration between Superwise and Datadog will enable Superwise’s model metrics and incidents integration streamline results of our out-of-the-box model metrics, including drift, activity, incidents, and any custom metrics you configure, directly into Datadog's dashboard.

What inspired you to submit this pull request?
Creating a new collaboration between Superwise and Datadog.

### Review checklist
- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X ] Git history is clean
- [ X] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
### Additional Notes

### CODE REVIEW

File: superwise/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Troubleshooting
+
+Need help? Contact [Superwise support][4].
+
+
+[1]: https://www.superwise.ai/
+[2]: https://portal.superwise.ai/
+[3]: https://github.com/DataDog/integrations-core/blob/master/check/metadata.csv
+[4]: https://docs.superwise.ai
```
Comment (anawishnoff) [inline superwise/README.md:57]: Is there a support email that can be used here? Otherwise, line 51 should be changed to something along the lines of "Need help? Take a look at our docs"

File: superwise/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/1.png)
+
+2. Click on Create a new channel and select Datadog Integration.
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/2.png)
+
+3. Input your Datadog API Key and Application key and click on the Test button. The Test button will send a dummy request to your Datadog account to validate the integration. You should get a success message in Superwise . To finish the setup click “Create channel”.
+
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/6.png)
+
+3. That's it, Now you can see a widget of new integration with Datdog and that it (:
```
Comment (anawishnoff) [inline superwise/README.md:27]:
```suggestion
3. That's it, now you can see the new Datadog integration widget.
```

File: superwise/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Once a user configures the Datadog integration in Superwise, standard model metrics are sent to Datadog and users get model observability dashboards within Datadog. Users can also configure any specific model metric and incident policy and send them to Datadog for model observability tailored to their business context.
+
+1. Go to [Superwise portal][2] and Select the integration tab.
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/1.png)
+
+2. Click on Create a new channel and select Datadog Integration.
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/2.png)
+
+3. Input your Datadog API Key and Application key and click on the Test button. The Test button will send a dummy request to your Datadog account to validate the integration. You should get a success message in Superwise . To finish the setup click “Create channel”.
```
Comment (anawishnoff) [inline superwise/README.md:23]:
```suggestion
3. Input your Datadog API Key and Application key and click on the Test button. The Test button will send a dummy request to your Datadog account to validate the integration. You should get a success message in Superwise. To finish the setup click “Create channel”.
```

File: superwise/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/6.png)
+
+3. That's it, Now you can see a widget of new integration with Datdog and that it (:
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/3.png)
+
+ ### Validation
+Go to the metric explorer section of the Datadog app to verify the integration between Superwise and Datadog,
+Superwise sent the superwise.integration.test metric to Datadog to validate the integration.
```
Comment (anawishnoff) [inline superwise/README.md:32]:
```suggestion
Within Datadog, go to the Metrics Explorer and search for the superwise.integration.test metric to verify that the integration between Superwise and Datadog was successful. 
```

File: .azure-pipelines/all.yml [hunk 5]
```diff
@@ -21,18 +21,18 @@ resources:
   - container: dd_agent
     image: gcr.io/datadoghq/agent:latest
     ports:
-      - 8127:8126
+    - 8127:8126
```
Comment (sarah-witt) [inline .azure-pipelines/all.yml:24]: Can you revert the changes in this file? They are not needed.

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -7,6 +7,7 @@
 
 # Community partners
 /algorithmia/                            @koverholt support@algorithmia.io
+/superwise/                            @oryan-omer support@superwise.ai
```
Comment (sarah-witt) [inline .github/CODEOWNERS:10]:
```suggestion
/algorithmia/                            @koverholt support@algorithmia.io
/superwise/                              @oryan-omer support@superwise.ai
```
nit- let's keep the codeowners formatting consistent

File: superwise/README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/2.png)
+
+3. Input your Datadog API Key and Application key and click on the Test button. The Test button will send a dummy request to your Datadog account to validate the integration. You should get a success message in Superwise . To finish the setup click “Create channel”.
+
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/6.png)
+
+3. That's it, Now you can see a widget of new integration with Datdog and that it (:
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/3.png)
+
+ ### Validation
```
Comment (sarah-witt) [inline superwise/README.md:30]:
```suggestion
### Validation
```

File: superwise/metadata.csv [hunk 8]
```diff
@@ -0,0 +1,5 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+superwise.metric.overall.data_drift,gauge,,day,,data drift,0,superwise,
+superwise.metric.overall.weighted_data_drift,gauge,,day,,weighted data drift,0,superwise,
+superwise.metric.overall.quantity,gauge,,day,,quantity,0,superwise,
+
```
Comment (sarah-witt) [inline superwise/metadata.csv:5]: Are these all of the supported metrics? The readme mentions activity, incident, and model metrics.

File: superwise/assets/dashboards/superwise.json [hunk 9]
```diff
@@ -0,0 +1 @@
...ests":[{"formulas":[{"formula":"query1"}],"response_format":"timeseries","queries":[{"query":"max:superwise.metric.overall.quantity{*} by {model_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"area"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":3,"width":4,"height":3}},{"id":7366254108548012,"definition":{"title":"Models Input Drift From Baseline","title_size":"16","title_align":"left","time":{},"type":"toplist","requests":[{"formulas":[{"formula":"query1","limit":{"count":5,"order":"desc"}}],"conditional_formats":[{"comparator":"<","palette":"white_on_green","value":50},{"comparator":">=","palette":"white_on_red","value":80},{"comparator":">=","palette":"white_on_yellow","value":50}],"response_format":"scalar","queries":[{"query":"max:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"last"}]}]},"layout":{"x":8,"y":3,"width":4,"height":3}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"3rs-pu9-3be"}
```
Comment (sarah-witt) [inline superwise/assets/dashboards/superwise.json:1]: Can you update the description of the dashboard?

**Review by sarah-witt — CHANGES_REQUESTED** [review]
Hi @oryan-omer thanks for the PR! I left some initial comments

File: superwise/metadata.csv (hunk 8 above)
Comment (oryan-omer) [reply superwise/metadata.csv:5]: These are the basic metrics.
The Superwise incident is not a metric, but it is equivalent to an incident in Datadog.

Comment (sarah-witt) [reply superwise/metadata.csv:5]: Sounds good! It's recommenced to list all known metrics here if possible

File: .azure-pipelines/all.yml (hunk 5 above)
Comment (sarah-witt) [reply .azure-pipelines/all.yml]: Can you remove the other changes in this file as well? They are only needed for integrations that have code in the repo.

**Comment by oryan-omer** [comment]
> **sarah-witt ** requested changes

All comments were fixed

File: .codecov.yml [hunk 10]
```diff
@@ -155,6 +155,10 @@ coverage:
         target: 75
         flags:
         - storm
+      Superwise:
+        target: 75
+        flags:
+        - superwise
```
Comment (sarah-witt) [inline .codecov.yml:161]:
```suggestion
```

File: .codecov.yml [hunk 11]
```diff
@@ -414,6 +418,11 @@ flags:
     paths:
     - storm/datadog_checks/storm
     - storm/tests
+  superwise:
+    carryforward: true
+    paths:
+    - superwise/datadog_checks/superwise
+    - superwise/tests
```
Comment (sarah-witt) [inline .codecov.yml:425]:
```suggestion
```

File: superwise/metadata.csv (hunk 8 above)
Comment (oryan-omer) [reply superwise/metadata.csv:5]: To start, these are the metrics (:

**Review by sarah-witt — COMMENTED** [review]
Hi @oryan-omer-  sorry, I think my more recent comments were hidden:
https://github.com/DataDog/integrations-extras/pull/1114#discussion_r778198439, but the changes in `codecov.yml` should be removed

File: .azure-pipelines/all.yml (hunk 5 above)
Comment (oryan-omer) [reply .azure-pipelines/all.yml]: I removed everything. (:

File: superwise/assets/dashboards/superwise.json (hunk 9 above)
Comment (oryan-omer) [reply superwise/assets/dashboards/superwise.json]: Fix

**Comment by oryan-omer** [comment]
> Hi @oryan-omer- sorry, I think my more recent comments were hidden: [#1114 (comment)](https://github.com/DataDog/integrations-extras/pull/1114#discussion_r778198439), but the changes in `codecov.yml` should be removed

Removed (:

File: .codecov.yml (hunk 11 above)
Comment (oryan-omer) [reply .codecov.yml]: Done

File: .codecov.yml (hunk 10 above)
Comment (oryan-omer) [reply .codecov.yml]: Removed

File: superwise/assets/dashboards/superwise.json [hunk 12]
```diff
@@ -0,0 +1 @@
...green","value":50},{"comparator":">=","palette":"white_on_red","value":80},{"comparator":">=","palette":"white_on_yellow","value":50}],"response_format":"scalar","queries":[{"query":"max:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"last"}]}]},"layout":{"x":0,"y":3,"width":4,"height":3}},{"id":1954347328001564,"definition":{"title":"Total # of Predictions - Trend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"response_format":"timeseries","queries":[{"query":"max:superwise.metric.overall.quantity{*} by {model_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"area"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":6,"width":4,"height":3}}]},"layout":{"x":8,"y":0,"width":4,"height":10}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"3rs-pu9-3be"}
```
Comment (maycmlee) [inline superwise/assets/dashboards/superwise.json:1]:
```suggestion
{"title":"Superwise","description":"## superwise overview dashboard\n\nThe model observability dashboard gives you out-of-the-box information for your active models, including activity statuses, drift levels, and open incidents detected for specific time intervals or filters. In addition, you can add any custom metrics or incidents to monitor for your specific use cases.\n\n- [https://docs.superwise.ai/docs](#docs)","widgets":[{"id":418963368099752,"definition":{"title":"Superwise","type":"group","background_color":"vivid_purple","show_title":true,"layout_type":"ordered","widgets":[{"id":6935498210192224,"definition":{"type":"note","content":"#### Model observability powered by\n![superwise](https://uploads-ssl.webflow.com/5f3245942591962669e600b0/61c6f497348387036a45152f_Superwise%20A%20logo%20black%20Small.png).  \nThe model observability dashboard gives you out-of-the-box information for your active models, including activity statuses, drift levels, and open incidents detected for specific time intervals or filters. In addition, you can add custom metrics and incidents to monitor your specific use cases.\n\nModel activity - Overview of model activity including the number of active models (models that had some production data during the filtered time), their activity (predictions) over time, and the total number of predictions during the filtered timeframe.\n\nDrift detection - Superwise's drift measures enable users to detect the model drift level in production relative to the baseline, for example, training dataset. The drift measure is scaled between 0-100 and is based on Superwise’s unique drift metric. Using the model input drift chart, users can identify which models are drifting and may require retraining.\n\nIncidents - Using incident widgets, users can easily see how many models currently have open incidents (violations of any monitoring policy configured), how incidents are being distributed among the different models, and drill down into the model incident details. The incidents that are included in this dashboard are only the ones configured explicitly to be sent to Datadog. To read more about how to define which incidents should be sent to Datadog, read more here.\nFor further customizations read our integration: [docs](https://docs.superwise.ai)","background_color":"white","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"has_padding":true},"layout":{"x":0,"y":0,"width":4,"height":6}}]},"layout":{"x":0,"y":0,"width":4,"height":7}},{"id":3345130583082,"definition":{"title":"Incidents","type":"group","background_color":"yellow","show_title":true,"layout_type":"ordered","widgets":[{"id":819512417324588,"definition":{"title":"# of Opened Incidents in the last time","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"conditional_formats":[{"comparator":">","palette":"white_on_red","value":5},{"comparator":">=","palette":"white_on_yellow","value":1},{"comparator":"<","palette":"white_on_green","value":1}],"response_format":"scalar","queries":[{"search":{"query":"severity:SEV-2"},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[]}]}],"autoscale":true,"precision":2},"layout":{"x":0,"y":0,"width":4,"height":3}},{"id":7223627866769632,"definition":{"title":"Incidents - Trend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"alias":"Models with Incidents","formula":"query1"}],"response_format":"timeseries","queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[]}],"style":{"palette":"purple","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":3,"width":4,"height":3}}]},"layout":{"x":4,"y":0,"width":4,"height":7}},{"id":5132323858991144,"definition":{"title":"Metrics","type":"group","background_color":"vivid_orange","show_title":true,"layout_type":"ordered","widgets":[{"id":2448869580205054,"definition":{"title":"Active Models","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"count_not_null(query1)"}],"conditional_formats":[{"comparator":">","palette":"white_on_green","value":1}],"response_format":"scalar","queries":[{"query":"sum:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"precision":2},"layout":{"x":0,"y":0,"width":2,"height":3}},{"id":2728068484701532,"definition":{"title":"Total # of Predictions","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"response_format":"scalar","queries":[{"query":"sum:superwise.metric.overall.quantity{*}","data_source":"metrics","name":"query1","aggregator":"sum"}]}],"autoscale":true,"precision":0},"layout":{"x":2,"y":0,"width":2,"height":3}},{"id":7366254108548012,"definition":{"title":"Models Input Drift From Baseline","title_size":"16","title_align":"left","type":"toplist","requests":[{"formulas":[{"formula":"query1","limit":{"count":5,"order":"desc"}}],"conditional_formats":[{"comparator":"<","palette":"white_on_green","value":50},{"comparator":">=","palette":"white_on_red","value":80},{"comparator":">=","palette":"white_on_yellow","value":50}],"response_format":"scalar","queries":[{"query":"max:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"last"}]}]},"layout":{"x":0,"y":3,"width":4,"height":3}},{"id":1954347328001564,"definition":{"title":"Total # of Predictions - Trend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"response_format":"timeseries","queries":[{"query":"max:superwise.metric.overall.quantity{*} by {model_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"area"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":6,"width":4,"height":3}}]},"layout":{"x":8,"y":0,"width":4,"height":10}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"3rs-pu9-3be"}
```

Comment (maycmlee) [reply superwise/assets/dashboards/superwise.json:1]: Made some small changes to the informational text.

File: superwise/README.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+Superwise’s model metrics and incidents integration streamline results of our out-of-the-box model metrics, including drift, activity, incidents, and any custom metrics you configure, directly into Datadog app. You’ll get an immediate overview of which models are misbehaving that can be tailored to any use case, logic, segmentation, threshold, and sensitivity.
+
+
+## Setup
+
+Once a user configures the Datadog integration in Superwise, standard model metrics are sent to Datadog and users get model observability dashboards within Datadog. Users can also configure any specific model metric and incident policy and send them to Datadog for model observability tailored to their business context.
+
+1. Go to [Superwise portal][2] and Select the integration tab.
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/1.png)
```
Comment (maycmlee) [inline superwise/README.md:18]: This image doesn't seem necessary.

File: superwise/README.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+2. Click on Create a new channel and select Datadog Integration.
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/2.png)
```
Comment (maycmlee) [inline superwise/README.md:21]: Can you crop this image so it only shows the "Add new integration" box?

File: superwise/README.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+3. Input your Datadog API Key and Application key and click on the Test button. The Test button will send a dummy request to your Datadog account to validate the integration. You should get a success message in Superwise. To finish the setup click “Create channel”.
+
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/6.png)
```
Comment (maycmlee) [inline superwise/README.md:25]: Please don't include API / APP key in screenshots. Make sure to rotate your keys since they are in the screenshot. Could you also crop the image to only show the "Add new Datadog channel" box?

File: superwise/README.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+3. That's it, now you can see the new Datadog integration widget.
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/3.png)
```
Comment (maycmlee) [inline superwise/README.md:28]: Could you crop image so it only shows the integration widget?

File: superwise/README.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+### Validation
+Go to the metric explorer section of the Datadog app to verify the integration between Superwise and Datadog,
+Within Datadog, go to the Metrics Explorer and search for the superwise.integration.test metric to verify that the integration between Superwise and Datadog was successful. 
+![Superwise Integration]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/4.png)
```
Comment (maycmlee) [inline superwise/README.md:33]: Could you crop this image to only show the metrics explorer and graph?

**Review by maycmlee — CHANGES_REQUESTED** [review]

File: superwise/README.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
+# Superwise Integration
+
+## Overview
+[Superwise][1]  is about solving model observability for high-scale ML operations.
+Superwise model observability gives you visibility and context into your models' behaviors, so you can easily express and monitor model risks as they relate to your business. With Superwise, data scientists, ML engineers, and business ops get intuitive model observability without the alert fatigue so you can be confident about your model management and focus on the fun things in life like building newer, better models.  
+    
+
+![Superwise Dashboard]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/5.png)  
+
+Superwise’s model metrics and incidents integration streamline results of our out-of-the-box model metrics, including drift, activity, incidents, and any custom metrics you configure, directly into Datadog app. You’ll get an immediate overview of which models are misbehaving that can be tailored to any use case, logic, segmentation, threshold, and sensitivity.
```
[the remark this answers is not in the document]
Comment (oryan-omer) [reply superwise/README.md:10]: In other words, the model can predict unwanted values, and the metrics we provide give him the ability to understand how to adjust the model to behave as expected

File: superwise/README.md (hunk 13 above)
Comment (oryan-omer) [reply superwise/README.md:18]: Removed

File: superwise/README.md (hunk 15 above)
Comment (oryan-omer) [reply superwise/README.md:25]: Done

File: superwise/README.md (hunk 16 above)
Comment (oryan-omer) [reply superwise/README.md:28]: Done

File: superwise/README.md (hunk 17 above)
Comment (oryan-omer) [reply superwise/README.md:33]: Done

File: superwise/README.md (hunk 14 above)
Comment (oryan-omer) [reply superwise/README.md:20]: Done

File: superwise/assets/dashboards/superwise.json [hunk 19]
```diff
@@ -0,0 +1 @@
...
```
Comment (anawishnoff) [inline superwise/assets/dashboards/superwise.json:1]: The dashboard is looking great, just had a few minor comments:

- In your text widget, regarding the last few sentences:

    _To read more about how to define which incidents should be sent to Datadog, **read more here**. For further customizations read our integration: **docs**_

    The "_**read more here**_" is missing a link. For the second sentence, changing it to "_For further dashboard customizations, see our integration documentation_" seems more clear.

- For the incidents widget, what is the "# of open incidents in the last time" referring to? Maybe "# of incidents opened in the last x hours" or something along those lines would make more sense (just a suggestion).

Comment (oryan-omer) [reply superwise/assets/dashboards/superwise.json]: I fixed both comments (👍🏻

File: .github/CODEOWNERS [hunk 20]
```diff
@@ -7,6 +7,7 @@
 
 # Community partners
 /algorithmia/                            @koverholt support@algorithmia.io
+/superwise/                              @oryan-omer support@superwise.ai
```
Comment (sarah-witt) [inline .github/CODEOWNERS:10]: Can we move this so the file remains in alphabetical order?

**Review by sarah-witt — COMMENTED** [review]
Looks good to me for agent-integrations, I just found one more small thing

File: .github/CODEOWNERS (hunk 20 above)
Comment (oryan-omer) [reply .github/CODEOWNERS]: Done

**Review by maycmlee — CHANGES_REQUESTED** [review]
Hi! Made some changes to the way the images are linked. Could you also remove 1.png since it's no longer being used?

File: superwise/README.md [hunk 21]
```diff
@@ -0,0 +1,55 @@
+# Superwise Integration
+
+## Overview
+[Superwise][1] provides model observability for high-scale machine learning (ML) operations.
+Superwise's model observability gives you visibility and context into your models' behaviors, so you can easily monitor model risks based on different use cases. With Superwise, data scientists, ML engineers, and business ops get model observability without alert fatigue, so you can be confident about your model management.  
+    
+
+![Superwise Dashboard]( https://raw.githubusercontent.com/DataDog/integrations-extras/master/superwise/images/5.png)  
+
+Superwise’s model metrics and incidents integration sends out-of-the-box model metrics, including drift, activity, incidents, and custom metrics, directly to Datadog. You’ll get an overview of which models are misbehaving that can be tailored to any use case, logic, segmentation, threshold, and sensitivity.
```
[the remark this answers is not in the document]
Comment (oryan-omer) [reply superwise/README.md:10]: Misbehaving means that the model does not predict the correct expected results, and this is evident by unexpected behavior on his metrics.

File: superwise/assets/dashboards/superwise.json (hunk 19 above)
Comment (anawishnoff) [reply superwise/assets/dashboards/superwise.json]: Sorry, for the "Last X hours" comment I meant that it would be helpful to provide a number - i.e. "in the last 24 hours"  or "in the last hour" or whatever the proper measurement is for that metric. Also, I'd recommend changing the last sentence in the text widget to:

 For further dashboard customizations, see our **integration documentation**

Thanks!

File: .github/CODEOWNERS [hunk 22]
```diff
@@ -95,6 +95,7 @@
 /sigsci/                                 @signalsciences info@signalsciences.com
 /shoreline/                              @ytnytn1 support@shoreline.io
 /sleuth/                                 @mtbnut pleal@sleuth.io
+/superwise/                              @oryan-omer support@superwise.ai
```
Comment (yzhan289) [inline .github/CODEOWNERS:98]: Small nit but this should go after `/storm/` not after `/sleuth/`.

File: superwise/assets/dashboards/superwise.json (hunk 19 above)
Comment (oryan-omer) [reply superwise/assets/dashboards/superwise.json]: Hi, the time depends on the resolution of the user, so maybe X is a good suggestion.
The last sentence in the widget already contains the sentence "For further dashboard customizations, see our integration documentation"

Comment (oryan-omer) [reply superwise/assets/dashboards/superwise.json]: @anawishnoff

File: .github/CODEOWNERS (hunk 22 above)
Comment (oryan-omer) [reply .github/CODEOWNERS]: Fixed @yzhan289

File: superwise/assets/dashboards/superwise.json (hunk 19 above)
Comment (anawishnoff) [reply superwise/assets/dashboards/superwise.json]: @oryan-omer  Ah I see - if it's based on the time since the last resolution of the user, changing it to "Number of incidents opened since last resolution" makes more sense to me. Apologies for the back and forth. 

For the last sentence, I was just suggesting to put the hyperlink over the bolded text below, rather than include the hyperlink over "docs":

For further dashboard customizations, see our **integration documentation**

**Review by maycmlee — CHANGES_REQUESTED** [review]

File: superwise/assets/dashboards/superwise.json (hunk 19 above)
Comment (oryan-omer) [reply superwise/assets/dashboards/superwise.json]: gotchya, Fixed (: @anawishnoff

**Review by anawishnoff — APPROVED** [review]

File: superwise/metadata.csv [hunk 23]
```diff
@@ -0,0 +1,5 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+superwise.metric.overall.data_drift,gauge,,day,,data drift,0,superwise,
+superwise.metric.overall.weighted_data_drift,gauge,,day,,weighted data drift,0,superwise,
+superwise.metric.overall.quantity,gauge,,day,,quantity,0,superwise,
```
[the remark this answers is not in the document]
Comment (oryan-omer) [reply superwise/metadata.csv:4]: Predictions.
Amount of records sent by users to our system.
Quantity is used in our system as well, and we want to stay aligned.

[the remark this answers is not in the document]
Comment (oryan-omer) [reply superwise/metadata.csv:4]: @maycmlee

File: superwise/metadata.csv [hunk 24]
```diff
@@ -0,0 +1,5 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+superwise.metric.overall.data_drift,gauge,,day,,Average distribution shifts across all features.,0,superwise,
+superwise.metric.overall.weighted_data_drift,gauge,,day,,Average distribution shifts across all features weighted by importance.,0,superwise,
+superwise.metric.overall.quantity,gauge,,day,,quantity,0,superwise,
```
Comment (maycmlee) [inline superwise/metadata.csv:4]:
```suggestion
superwise.metric.overall.quantity,gauge,,day,,Number of records sent.,0,superwise,
```

**Review by maycmlee — CHANGES_REQUESTED** [review]

File: superwise/metadata.csv (hunk 24 above)
Comment (oryan-omer) [reply superwise/metadata.csv]: Done @maycmlee

File: superwise/README.md [hunk 25]
```diff
@@ -0,0 +1,61 @@
...
+2. Click **Create a new channel** and select **Datadog**.
+
+![Superwise - Add new integration][4]
+
+3. Input your Datadog API Key and Application key, and click **Test**. A test request is sent to your Datadog account to validate the integration. If the request was successfully sent, there is a message in Superwise saying the test was delivered successfully. To finish the setup, click **Create channel**.
+
+![Superwise - Add new Datadog channel][5]
+
+3. That's it, now you can see the new Datadog integration widget.
+![Superwise Integration][6]
```
Comment (maycmlee) [inline superwise/README.md:27]:
```suggestion

![Superwise Integration][6]
```

File: superwise/README.md [hunk 26]
```diff
@@ -0,0 +1,61 @@
+# Superwise Integration
+
+## Overview
+[Superwise][1] provides model observability for high-scale machine learning (ML) operations.
+Superwise's model observability gives you visibility and context into your models' behaviors, so you can easily monitor model risks based on different use cases. With Superwise, data scientists, ML engineers, and business ops get model observability without alert fatigue, so you can be confident about your model management.  
+    
+![Superwise Dashboard][2]
+
+Superwise’s model metrics and incidents integration sends out-of-the-box model metrics, including drift, activity, incidents, and custom metrics, directly to Datadog. You’ll get an overview of which models are not predicting the expected results, which can be configured to any use case, logic, segmentation, threshold, and sensitivity.
```
Comment (maycmlee) [inline superwise/README.md:9]:
```suggestion
Superwise’s model metrics and incidents integration sends out-of-the-box model metrics, including drift, activity, incidents, and custom metrics, directly to Datadog. You get an overview of which models are not predicting the expected results, which can be configured to any use case, logic, segmentation, threshold, and sensitivity.
```

**Review by maycmlee — CHANGES_REQUESTED** [review]
Just two small changes. Otherwise, looks good for docs.
